scripts/combine/setupCombine.py

```diff
@@ -652,10 +652,6 @@ def assertSample(name, startsWith=["W", "Z"], excludeMatch=[]):
                                              )
         muRmuFPolVar_helper.add_theoryAgnostic_uncertainty()
 
-    if args.doStatOnly:
```

Comment:
I see you moved the "do stat only" due to the width fit, but then the mass fit where the width is a systematic background would be added also for "do stat only" this is not what we want

Reply:
Yeah, so here I would suggest adding one option to define the specific nuisance parameter of interest (so far it was the mass, of the Z or the W as appropriate, now it might be also the width, or potentially both for a simultaneous fit), and make sure that the stat only fit only sees those ones.